modular_skyrat/modules/contractor/code/items/misc.dm

```diff
@@ -5,8 +5,8 @@
 	info = {"<p>Welcome agent, congratulations on your new position as contractor. On top of your already assigned objectives,
 			this kit will provide you contracts to take on for TC payments.</p>
 
-			<p>Provided within, we give your specialist contractor space suit. It's even more compact, being able to fit into a pocket, and faster than the
-			Syndicate space suit available to you on the uplink. We also provide your chameleon jumpsuit and mask, both of which can be changed
+			<p>You likely already have your Contractor MODSuit equipped. It has a built in chameleon module, which only works when the MODSuit is undeployed,
```

Comment:
I'm pretty sure the chameleon modules requires your MODsuit to de deployed.
```suggestion
			<p>You likely already have your Contractor MODsuit equipped. It has a built in chameleon module, which only works when the MODsuit is deployed,
```

Reply:
That's intentional, it only works when undeployed.